libs/components/flyout/src/lib/modules/flyout/types/flyout-config.ts

```diff
@@ -6,26 +6,29 @@ import { SkyFlyoutPermalink } from './flyout-permalink';
  */
 export interface SkyFlyoutConfig {
   /**
-   * The HTML element ID (without the leading `#`) of the element that describes
+   * The HTML element ID of the element that describes
    * the flyout. This sets the flyout's `aria-describedby` attribute
-   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * to provide a text equivalent for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
    * The description typically includes text on the flyout but not on items that users
    * interact with, such as buttons and forms.
+   * For more information about the `aria-describedby` attribute, see the [WAI-ARIA definition](https://www.w3.org/TR/wai-aria/#aria-describedby).
    */
   ariaDescribedBy?: string;
 
   /**
    * The ARIA label for the flyout. This sets the flyouts's `aria-label` attribute
```

Comment:
```suggestion
   * The ARIA label for the flyout. This sets the flyouts's `aria-label` attribute to provide a text equivalent for screen readers 
```
Just adding the same text we added elsewhere for consistency.